Library/Formula/libgraphqlparser.rb

```diff
@@ -0,0 +1,39 @@
+require "open3"
+
+class Libgraphqlparser < Formula
+  desc "GraphQL query parser in C++ with C and C++ APIs"
+  homepage "https://github.com/graphql/libgraphqlparser"
+  url "https://github.com/graphql/libgraphqlparser/archive/v0.1.0.tar.gz"
+  sha256 "e4a12bba2f6c2db1b0528db5a0a3283fc5380ac5cadb8036793054cb7c9dd3ad"
+
+  depends_on "cmake" => :build
+  depends_on "bison" => :recommended
+  depends_on "flex"  => :recommended
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    bin.install "dump_json_ast"
```

Comment:
> looks like there is no `make install`

There is in the latest master, but not in version 0.1.0.  Maybe we should ask them to make a release so that it can more easily be packaged.